# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsContentPolicyUtils.h
# Commit: 98a751f6f41d
# Full Hash: 98a751f6f41d4e8d23f38aa0aa1d483f83cd0bda
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2023-01-19 04:46:07
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Implement csp for Module Workers; r=evilpie,asuth,rpl,ckerschb
#   
#   Depends on D155691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156102
# ==============================================================================

diff -r 78addb6af24d -r 98a751f6f41d dom/base/nsContentPolicyUtils.h
--- a/dom/base/nsContentPolicyUtils.h	Wed Jan 18 13:46:32 2023 +0000
+++ b/dom/base/nsContentPolicyUtils.h	Wed Jan 18 13:46:33 2023 +0000
@@ -144,8 +144,10 @@
     CASE_RETURN(TYPE_INTERNAL_FRAME_MESSAGEMANAGER_SCRIPT);
     CASE_RETURN(TYPE_INTERNAL_FETCH_PRELOAD);
     CASE_RETURN(TYPE_UA_FONT);
+    CASE_RETURN(TYPE_INTERNAL_WORKER_STATIC_MODULE);
     CASE_RETURN(TYPE_PROXIED_WEBRTC_MEDIA);
     CASE_RETURN(TYPE_WEB_IDENTITY);
+    CASE_RETURN(TYPE_END);
     case nsIContentPolicy::TYPE_INVALID:
       break;
       // Do not add default: so that compilers can catch the missing case.